core/src/main/java/io/undertow/server/protocol/http2/Http2ServerConnection.java

```diff
@@ -152,6 +152,10 @@ public SSLSession getSslSession() {
         return channel.getSslSession();
     }
 
+    public Http2Channel getChannel() {
```

Comment:
There is no need to add this method. Just invoking addCloseListener in the connection will add the listener to the channel.